test/ejabberd_tests/tests/muc_light_SUITE.erl

```diff
@@ -965,6 +943,14 @@ info_iq_verify_fun(AffUsers, Version, ConfigKVBin) ->
             verify_config(ConfigurationEl, ConfigKVBin)
     end.
 
+-spec verify_user_has_one_room() -> fun((User ::escalus:client()) -> any()).
+verify_user_has_one_room() ->
```

Comment:
It would more readable if the function took `User` as a parameter. In this case you don't have to return `fun` but simply do what you need.

Such a function can be passed to escalus:story like this: `fun verify_user_has_one_room/1`